section_start_rules.js

```diff
@@ -0,0 +1,147 @@
+var fs = require('fs');
+var cheerio = require('cheerio');
+
+// --------------------------------------------  LOCAL DECLARATIONS
+var file = process.argv[2];
+
+// read in json
+var rulesjson = require(process.argv[3]);
+
+// Read in file contents
+fs.readFile(file, function editContent (err, contents) {
+  $ = cheerio.load(contents, {
+          xmlMode: true
+        });
+
+
+// -------------------------------------------- FUNCTIONS
+function processRule(rule, section_types) {
+  // Are our values getting here?
+  // console.log("rule_name : " + rule['rule_name']);
+  // console.log("ss_name : " + rule['ss_name']);
+  // console.log("section_required : " + rule['section_required']);
+  // console.log("insert_before : " + rule['insert_before']);
+  // console.log("multiple : " + rule['multiple']);
+  // console.log("styles : " + rule['styles']);
+  // console.log("position : " + rule['position']);
+  // console.log("optional_heading_styles : " + rule['optional_heading_styles']);
+  // console.log("first_child : " + rule['first_child']);
+  // console.log("first_child_text : " + rule['first_child_text']);
+  // console.log("first_child_match : " + rule['first_child_match']);
+  // console.log("previous_sibling : " + rule['previous_sibling']);
+  // console.log("previous_until : " + rule['previous_until']);
+  // console.log("required_styles : " + rule['required_styles']);
+  // console.log("previous_until_stop : " + rule['previous_until_stop']);
+  // console.log("last : " + rule['last']);
+  // console.log(section_types['frontmatter_sections']);
+  // console.log(file);
+
+  // Get to work!:
+
+}
+
+// Constructor for section start rules
+function Rule(key, values_hash, rule_number, section_types) {
+  if (rule_number < 9) {
+    criteria_count = '0' + rule_number;
+  } else {
+    criteria_count = rule_number;
+  }
+
+  this.rule_name = key + "_" + criteria_count;
+  this.ss_name = key;
+  if (values_hash.hasOwnProperty('section_required')) {
+    this.section_required = 'True';
```

Comment:
Curious why you went with a string version instead of a boolean? Standard markup would be: this.section_required = true; (and similarly for false)